Sources/TSPL/TSPL.docc/ReferenceManual/SummaryOfTheGrammar.md

```diff
@@ -0,0 +1,1751 @@
+# Summary of the Grammar
```

Comment:
Should there be a comment of some sort for this file that its contents need to be updated / duplicated along with grammar changes on the individual pages?

Reply:
Good point.

Reply:
Agreed.  I added a comment in d73aab87 at the top of the summary chapter.

It would be nice to automate that process at some point in the future.  Filed https://github.com/apple/swift-book/issues/61 to track that.